docs/docs/checks.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Checks
+---
+
+# Checks
+
+Marathon already has the concept of [health checks]({{ site.baseurl }}/docs/health-checks.html) and [readiness checks]({{ site.baseurl }}/docs/readiness-checks.html), which either monitor the health of an application or provide an indication that the service is ready.  Both of these features in Marathon will indicate that the app/pod is "in deployment".
+
+Checks provide an abstract way to check on a task in Mesos providing values back via tasks and podstatus which are *not* interpreted by Marathon.   They are intended to be consumed by services external to Marathon which will take appropriate action on them.   You can think of them as a health check without a kill policy.  The most significant aspect of them is that Marathon does not evaluate anything about the return values of a check.
```

Comment:
I would link mesos docs here

Reply:
Using terminology "already has the concept" sounds like terminology for a changelog. I think this page should be written from the perspective of what is, not what was.

Reply:
@timcharper this consistent with [readiness check docs](https://mesosphere.github.io/marathon/docs/readiness-checks.html) and helps to set context for user against features they are already familiar with... if they are not... links are provided.

Reply:
Okay. It's just that it will read weird a year in the future perhaps?